src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/RichTextEditorTextArea.kt

```diff
@@ -0,0 +1,65 @@
+package com.atlassian.performance.tools.jiraactions.api.page
+
+import com.atlassian.performance.tools.jiraactions.Patience
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+import org.openqa.selenium.By
+import org.openqa.selenium.JavascriptExecutor
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.WebElement
+import org.openqa.selenium.support.ui.ExpectedConditions
+import org.openqa.selenium.support.ui.ExpectedConditions.*
+import java.time.Duration
+import org.openqa.selenium.support.ui.WebDriverWait
+
+
+
+/**
+ * Helper class for Rich Text Editor enabled Textarea
```

Comment:
Documentation should explain what does the class represent or do.
"Helper" doesn't mean much (same as "util", "bean", "thing", "object").
And we already know it's a class, the code is right there and it is also not the most interesting aspect.

IMHO the most useful aspects are:
* represents the [Jira Rich Text Editor]( https://confluence.atlassian.com/adminjiraserver/rich-text-editing-938847886.html)
* it falls back to the plain text editor if RTE is disabled